stable/zeppelin/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Web-based notebook that enables data-driven, interactive data analytics and collaborative documents with SQL, Scala and more.
 name: zeppelin
-version: 1.0.1
+version: 1.0.2
```

Comment:
since you are adding a new feature, I would say to bump to `1.1.0`

Reply:
Good point. I have done it.